src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.Windows.cs

```diff
@@ -70,5 +89,29 @@ private static Architecture Map(int processorArchitecture)
                     return Architecture.X86;
             }
         }
+
+        private static Architecture MapMachineConstant(ushort processMachine)
+        {
+            switch (processMachine)
+            {
+                case 0x01C0: // IMAGE_FILE_MACHINE_ARM
+                case 0x01C2: // IMAGE_FILE_MACHINE_THUMB
```

Comment:
```suggestion
```
These are Windows CE architectures.

Reply:
`IMAGE_FILE_MACHINE_ARM` is handled in `src/coreclr/pal/inc/rt/ntimage.h`. Should we delete that?

Reply:
src/coreclr/pal/inc/rt/ntimage.h is a verbatim copy of the Windows OS header with PE file structure definitions. It cpntains a lot stuff that we do not use. I do not think it is worth it to try to trim. It will make updates harder.

Reply:
> src/coreclr/pal/inc/rt/ntimage.h is a verbatim copy of the Windows OS header with PE file structure definitions.

Maybe add this as a comment near the top of the file.

Reply:
Added.